doc/api/worker_threads.md

```diff
@@ -203,6 +203,11 @@ input of [`port.postMessage()`][].
 Listeners on this event will receive a clone of the `value` parameter as passed
 to `postMessage()` and no further arguments.
 
+When there are `'message'` event listeners and `.start()` is not called
+explicitly, and the Web-based `.onmessage` property is not used, no messages
```

Comment:
`web-based`?

Reply:
(Maybe `web-baed` should be removed? It seems likely to be misunderstood. I *think* we just mean "modeled on the analogous web-based API" but people are likely to interpret as more like...something to do with the web directly? That's what I did when I read this for a few seconds.

Reply:
(Both comments above are non-blocking.)

Reply:
I think removing `web-based` would be a bit confusing, given that we don’t currently document `onmessage` ourselves.